### PR TITLE
Add CANdy-Test genesis files

### PR DESCRIPTION
**Pull request by WadeBarnes**
Signed-off-by: Wade Barnes <wade@neoterictech.ca>

### CODE REVIEW

File: CANdy/test/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+[Hyperledger Indy-Node/noeud](https://github.com/hyperledger/indy-node) 
+
+The source files and configurations used to establish this network can be found in the repository - Les fichiers sources et les configurations utilisés pour établir ce réseau sont disponibles dans le dépôt suivant :
+
+[CQEN-QDCE/Candy](https://github.com/CQEN-QDCE/Candy)
+
+## [pool_transactions_genesis](./pool_transactions_genesis)
+
+This file contains the set of genesis nodes for the CANdy-Test network.  This is the genesis file to use when establishing a client/agent connection to the network. - Ce fichier contient l'ensemble des nœuds Genesis du registre du réseau de la couche test CANdy. Il s'agit du fichier Genesis à utiliser lors de l'établissement d'une connexion client/agent au réseau :
```
Comment (PhilippeBourque) [inline CANdy/test/README.md]: "nœuds Genesis" in french is "nœuds d'origine".
"fichier Genesis" in French is "fichier d'origine".

File: CANdy/test/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+## [domain_transactions_genesis](./domain_transactions_genesis)
+
+This file contains the set of genesis Trustees and Stewards for the CANdy-Test network. - Ce fichier contient l'ensemble du fichier Genesis Trustees et Stewards du registre du réseau de la couche test CANdy.
```
Comment (PhilippeBourque) [inline CANdy/test/README.md]: "Ce fichier contient l'ensemble du fichier Genesis Trustees et Stewards du registre du réseau de la couche test CANdy." should be written "Ce fichier contient les informations des Administrateurs de confiance et des Intendants d'origine pour la couche Test du réseau CANdy." in French.

File: CANdy/test/domain_transactions_genesis [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{"reqSignature":{},"txn":{"data":{"alias":"Wade Barnes","dest":"MNHGnsZF4KS6wykfRKUuLS","role":"0","verkey":"C6qcDRQiswUGpWK4FgDsHtaVUVjR4nRDYurZAivoetPZ"},"metadata":{},"type":"1"},"txnMetadata":{"seqNo":1},"ver":"1"}
+{"reqSignature":{},"txn":{"data":{"alias":"Stephen Curran","dest":"7aMWWD5d6dyb9FHY6RisAn","role":"0","verkey":"~Wg2CDurCSNyVfMypkK66Xz"},"metadata":{},"type":"1"},"txnMetadata":{"seqNo":2},"ver":"1"}
+{"reqSignature":{},"txn":{"data":{"alias":"Clecio Varjao","dest":"HL7vUSZi2fKzG3WsUo1CBf","role":"0","verkey":"~PJKwjd44UBHQ7KNc9Wtqav"},"metadata":{},"type":"1"},"txnMetadata":{"seqNo":3},"ver":"1"}
+{"reqSignature":{},"txn":{"data":{"alias":"John Jordan","dest":"iEvQcW7oFdsgNYJUwTRm6","role":"0","verkey":"~Srp7RBqd4PKqcdyWiigGYU"},"metadata":{},"type":"1"},"txnMetadata":{"seqNo":4},"ver":"1"}
+{"reqSignature":{},"txn":{"data":{"dest":"5pZSaMbNNTaarwjivRdz1p","role":"2","verkey":"3dNRSiSdxDxEvESBmW76azbUwSC8P4PeD655JXG8sS9m"},"metadata":{"from":"iEvQcW7oFdsgNYJUwTRm6"},"type":"1"},"txnMetadata":{"seqNo":5},"ver":"1"}
```
Comment (PhilippeBourque) [inline CANdy/test/domain_transactions_genesis:5]: Should we keep in the ReadMe file a list of provider (AWS, Azure, GGC, IBM, etc..) for each Stewards, just to have a quick look of the current network and keep in mind that we need diversity of provider when we add new node to the network? It's just a suggestion, maybe that is something we need to address with the Board eventually.

**Review by PhilippeBourque — CHANGES_REQUESTED** [review]
Bonjour Wades, 

I make some minor corrections for the French translation.

Merci! :) 

--------------------------------------------------------------------------------------------
For futur works, there is a quick translation table that we just finish with our tech specialists and translators : 

Board member = Membre du conseil
CANdy (PanCanadian Indy Ledger and Digital Identity) = CANdy (Registre de preuves Indy Pancanadien pour l’identité numérique)  
DID (Decentralized identifier) = Identifiant décentralisé
Endorser = Accréditeur
Governance framework = Cadre de gouvernance
Governing Board = Conseil de gouvernance 
Ledger = Registre de preuves
Registered entity (issuers) = Représentant autorisé
Steward = Intendant
Trustee = Administrateur de confiance
VC = (verifiable credential) = Attestation vérifiable

We will share it to the DevOps Cops and Governance Board this week. 
I will also try to find a great place to put it on the CANdy github repo.

**Comment by WadeBarnes** [comment]
@PhilippeBourque, Thanks for the review.  Updates made.

**Review by PhilippeBourque — APPROVED** [review]
Seems good for me.

**Review by PhilippeBourque — APPROVED** [review]
Good for me. 
(as you see, I'm still learning how github works - I thought I had the writing rights, so I make another Review, sorry for that)

**Comment by WadeBarnes** [comment]
Weird.  It should merge automatically if you approve the PR in the review changes section, but it looks like you've already approved it.  Perhaps try again.

**Review by PhilippeBourque — APPROVED** [review]
Approve.

**Comment by PhilippeBourque** [comment]
It's look like I'm not autorize to Write on this repo.

**Comment by WadeBarnes** [comment]
It looks like we'll need @swcurran or @watkinspd to approve the PR for it to be merged.

**Review by swcurran — APPROVED** [review]
Per approval by Wade/Philippe.

**Pull request merged**